pkg/util/mount/nsenter_mount.go

```diff
@@ -58,7 +60,16 @@ type NsenterMounter struct {
 	paths map[string]string
 }
 
-func NewNsenterMounter() *NsenterMounter {
+func NewNsenterMounter() (*NsenterMounter, error) {
+	// check if we have CAP_SYS_ADMIN to setgroup properly
+	pid, err := capability.NewPid(os.Getpid())
+	if err != nil {
+		return nil, err
+	}
+	if !pid.Get(capability.EFFECTIVE, capability.CAP_SYS_ADMIN) {
+		return nil, fmt.Errorf("Kubelet needs to run in a privileged container")
```

Comment:
`Kubelet needs the CAP_SYS_ADMIN capability, which means you probably need to be root, or run in a privileged container`